src/src.pro

```diff
@@ -5,7 +5,6 @@ PKGCONFIG += usb_moded
 
 QT += dbus
 QT -= gui
-QMAKE_CXXFLAGS += -Wno-unused-parameter -Wno-psabi
```

Comment:
Just observation regarding -Wno-unused-parameter: there is e.g. QUsbModed::onServiceUnregistered() with parameter that was used / not depending on what kindof DEBUG_() macro got defined. Anyway, not relevant after debugging changes.

Reply:
This was the last commit so at that point there was no DEBUG_ macro anymore.